### ISSUE TITLE
What is this project bringing over anira ? 

### ISSUE THREAD

**Issue by mchagneux** [Torsion-Audio/nn-inference-template#2]
Hi all, 

Just watched the ADC talk and came skimming through this repo to understand it a bit better. I see that the guts of the template is the anira project from TU-Studio and I'm not 100% sure if this repo just brings a pretty UI or if there's something more to it ? Just asking because anira seems to already provide a plugin example, and receives update sfaster than this repo so I'm wondering which one I should fork :) 

Thanks in advance!

**Comment by faressc** [comment]
Hi there,
Thanks for reaching out. You are right, basically this repo just brings a pretty UI to the anira-juce-example and shows how anira can be used in a separate repository. We plan to keep the nn-inference-template up to date with anira, but the reason it is not right now is that our Apple Developer ID from the university has expired. An update would leave the binaries unnotarized in the next release. As soon as our Apple ID is renewed, we'll update the repo.
In short, we strongly recommend to use the latest version of anira because of the many improvements.
Cheers,
Fares

**Comment by mchagneux** [comment]
Ok, very clear thank you for answering !

**Comment by faressc** [comment]
We finally got the developer license again. So the notarization works again and the nn-inference-template is updated to the latest anira version.
